docker/actinia-core-tests/actinia-test-noauth.cfg

```
@@ -0,0 +1,44 @@
+[GRASS]
+grass_gis_start_script = /usr/local/bin/grass
+grass_database = /actinia_core/grassdb
+grass_user_database = /actinia_core/userdata
+grass_tmp_database = /actinia_core/workspace/temp_db
+grass_resource_dir = /actinia_core/resources
+grass_addon_path = /root/.grass8/addons/
+grass_gis_base = /usr/local/grass
+grass_modules_xml_path = /usr/local/grass/gui/wxpython/xml/module_items.xml
+grass_default_location = nc_spm_08
+
+[API]
+plugins = []
+force_https_urls = True
+authentication = False
+
+[LIMITS]
+max_cell_limit = 22500000
+process_time_limt = 60
+process_num_limit = 20
+number_of_workers = 3
+
+[REDIS]
+redis_server_url = localhost
+redis_server_port = 6379
+redis_queue_server_url = localhost
+redis_queue_server_port = 6379
+worker_queue_name = actinia_job
+worker_logfile = /actinia_core/workspace/tmp/actinia_worker_test.log
+
+[LOGGING]
+log_interface = fluentd
+log_fluent_host = fluentd
+log_fluent_port = 24224
+log_level = 1
+
+[MISC]
+tmp_workdir = /actinia_core/workspace/tmp
+download_cache = /actinia_core/workspace/download_cache
+secret_key = token_signing_key_changeme
+
+[MANAGEMENT]
+default_user = user
+default_user_group = group
```

It would be nice if this new config file would follow the "new" config file example with slightly different order of [GRASS] section and omitting of defaults like [here](https://github.com/actinia-org/actinia-docker/blob/main/actinia-alpine/actinia.cfg)

Example:

```suggestion
[GRASS]
grass_database = /actinia_core/grassdb
grass_user_database = /actinia_core/userdata
grass_tmp_database = /actinia_core/workspace/temp_db
grass_resource_dir = /actinia_core/resources
grass_gis_base = /usr/local/grass
grass_gis_start_script = /usr/local/bin/grass
grass_addon_path = /root/.grass8/addons/

[API]
plugins = []
force_https_urls = True
authentication = False

[REDIS]
redis_server_url = localhost
redis_server_port = 6379

[LOGGING]
log_level = 1

[MISC]
tmp_workdir = /actinia_core/workspace/tmp
download_cache = /actinia_core/workspace/download_cache
secret_key = token_signing_key_changeme
```

---

I have now changed it